eng/templates/stages/release.yml

```
@@ -401,6 +405,8 @@ stages:
             echo "  --title $title"
             echo "  --body $body"
             echo "  --targetBranch $target_branch"
+            echo "  --resolvedSdkArtifactFileName $resolvedSdkArtifactFileName"
+            echo "  --resolvedSourceBuiltArtifactsFileName $resolvedSourceBuiltArtifactsFileName"
```

I think this is no longer `--resolved..`

---

There is already a variable called sourceBuiltArtifactsFileName.

---

Yeah but we need to fix it here too?

```suggestion
            echo "  --sdkArtifactFileName $resolvedSdkArtifactFileName"
            echo "  --sourceBuiltArtifactsFileName $resolvedSourceBuiltArtifactsFileName"
```

---

Done, thank you :)